alv-portal-ui/src/app/admin/system-notifications/system-notifications.component.ts

```
@@ -0,0 +1,90 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractSubscriber } from '../../core/abstract-subscriber';
+import { Subject } from 'rxjs';
+import { SystemNotificationDto } from '../../shared/backend-services/system-notifications/system-notification.types';
+import { SystemNotificationRepository } from '../../shared/backend-services/system-notifications/system-notification-repository';
+import { ModalService } from '../../shared/layout/modal/modal.service';
+import { SystemNotificationEditModalComponent } from './edit-modal/system-notification-edit-modal.component';
+import { SystemNotificationCreateModalComponent } from './create-modal/system-notification-create-modal.component';
+import { NotificationsService } from '../../core/notifications.service';
+import { ConfirmModalConfig } from '../../shared/layout/modal/confirm-modal/confirm-modal-config.model';
+
+@Component({
+  selector: 'alv-system-notifications',
+  templateUrl: './system-notifications.component.html',
+})
+export class SystemNotificationsComponent extends AbstractSubscriber implements OnInit {
+
+  systemNotifications$ = new Subject<SystemNotificationDto[]>();
+
+  constructor(private systemNotificationRepository: SystemNotificationRepository,
+              private modalService: ModalService,
+              private notificationsService: NotificationsService) {
+    super();
+  }
+
+  ngOnInit(): void {
+    this.loadSystemNotifications();
+  }
+
+  openCreateModal() {
+    const createModalRef = this.modalService.openLarge(SystemNotificationCreateModalComponent);
+    createModalRef.result
+      .then(() => {
+        this.loadSystemNotifications();
+        this.notificationsService.success('portal.admin.system-notifications.notifications.create.success', false);
+      })
+      .catch(() => {
+      });
```

why do we swallow the error here?